openapi.yaml

```diff
@@ -779,10 +1332,46 @@ components:
       scheme: bearer
       bearerFormat: JWT
   schemas:
+    UserId:
```

Comment:
Why create another layer of reference from UserId -> uid?

Reply:
Yeah, no real reason. I have renamed uid to UserId